eZ/Publish/Core/Persistence/Cache/ContentLanguageHandler.php

```diff
@@ -29,10 +29,10 @@ class ContentLanguageHandler extends AbstractInMemoryPersistenceHandler implemen
     protected function init(): void
     {
         $this->getTags = static function (Language $language) { return ['language-' . $language->id]; };
-        $this->getKeys = static function (Language $language) {
+        $this->getKeys = function (Language $language) {
```

Comment:
Why removing `static`?

Reply:
to use the escapeForCacheKey function we need to change from a static function to normal